src/Demos/13-actor-example.fsx

```diff
@@ -0,0 +1,102 @@
+﻿#load "credentials.fsx"
+#load "lib/collections.fsx"
+
+open MBrace
+open MBrace.Azure
+open MBrace.Azure.Client
+open MBrace.Azure.Runtime
+open MBrace.Streams
+open MBrace.Workflows
+open Nessos.Streams
+
+(**
+ This tutorial demonstrates a simple actor library implemented on top of MBrace
+ 
+ Before running, edit credentials.fsx to enter your connection strings.
+**)
+
+// First connect to the cluster
+let cluster = Runtime.GetHandle(config)
+
+
+/// Actor message inbox
+type Mailbox<'T> = IReceivePort<'T>
+/// Actor reply channel
+type ReplyChannel<'T> = ISendPort<'T>
+
+/// Cloud actor instance
+type CloudActor<'T> =
+    {
+        Sender : ISendPort<'T>
+        Task : ICloudTask<unit>
+        CancellationTokenSource : ICloudCancellationTokenSource
+    }
+with
```

Comment:
I recommend removing "with" in F# code, just align the members correctly (as you have already done)